packages/fullstack/src/render.rs

```diff
@@ -152,6 +153,122 @@ impl SsrRendererPool {
                 });
                 let (freshness, html) = rx.await.unwrap()?;
 
+                Ok((freshness, html))
+            }
+        }
+        #[cfg(target_arch = "wasm32")]
```

Comment:
This code has not been tested with changes to pass compilation. Is there a better way to write it?

Reply:
It looks like there is a lot of duplication between the wasm and not wasm implementations. Is it possible to pull out our own version of spawn_local that tries to use `get_local_pool().spawn_pinned` on most platforms but uses `spawn_local` directly on wasm?